lib/models/report.dart

```diff
@@ -161,6 +168,14 @@ class Report {
     data['app_language'] = app_language;
     return data;
   }
+
+  LatLng getLocation() {
```

Comment:
There's actually a third scenario we need to consider with location_choice: sometimes it's marked as 'missing' for old reports that didn't include location data. In those cases, it might be cool to just return null.
It should be interesting to force location_choice == current , location_choice == selected and else null.

Reply:
Okey! I've added the third scenario, but should we return null or (0,0) like when gps is disabled? If we return null, I'd have to disable the minimap and location coordinates inside of "My Reports"

Not related with your comment, but related with this method: I also considered using Position from geolocator instead of LatLng from GoogleMaps as we talked, but there were 2 drawbacks:
- Position requires the method to be async instead of sync
- Position required more parameters to be set (altitude, timestamp...), while LatLng is just latitude and longitude, which is what we need

Reply:
Conceptually speaking, in my opinion, it makes sense to return null instead of latlon 0,0.

Reply:
I've made it nullable and I think it looks good. If location is null, I just don't show the minimap, coordinates and displayCity like in this screenshot:

![image](https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/assets/157690872/9e06ed71-6a5a-4a2f-858f-79478f2e8941)